salt/apache/init.sls

```diff
@@ -33,6 +33,23 @@ apache2:
     - require:
       - pkg: apache2
 
+# Uploads custom Apache defaults page and configuration
+  file.managed:
+    - names: 
+      - /var/www/html/error_page.html:
+        - source: salt://apache/files/docs/error_page.html
+      - /etc/apache2/sites-avaliable/default.conf:
+        - source: salt://apache/files/conf/default.conf
```

Comment:
Since default.conf is installed in sites-available, we should put this in `files/sites` instead of `files/conf`.